app/services/reporting/emailable_report.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
```

Comment:
there aren't any strings in this file so we can probably remove?
```suggestion
```

Reply:
RubyMine likes to add that by default and I've just grown used to it. Good catch!